NetworkingSampleApp/NetworkingSampleApp/API/Routers/SampleUploadRouter.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SampleUploadRouter.swift
+//  NetworkingSampleApp
+//
+//  Created by Tony Ngo on 12.06.2023.
+//
+
+import Foundation
+import Networking
+import UniformTypeIdentifiers
+
+enum SampleUploadRouter: Requestable {
+    case image
+    case file(URL)
+
+    var baseURL: URL {
+        fatalError("Provide your API base URL for upload")
+    }
+
+    var headers: [String : String]? {
+        switch self {
+        case .image:
+            return ["Content-Type": "image/png"]
+        case let .file(url):
+            return ["Content-Type": url.mimeType]
+        }
+    }
+
+    var path: String {
+        fatalError("Provide your API endpoint path for upload")
```

Comment:
do we want a functional sample? or just code wise?

Reply:
If it's desired I can add httpbin.org which I was using in dev.

Reply:
got it, I wouldn't say it's needed for just demonstration - but final call is not on me

Reply:
I would add some default value, I don't feel it's a good idea throw fatal error when showcasing sample solution